mods/ts/rules/gdi-infantry.yaml

```diff
@@ -78,7 +80,7 @@ JUMPJET:
 	Voiced:
 		VoiceSet: JumpJet
 	Mobile:
-		Speed: 56
+		Speed: 71
```

Comment:
We seem to use `56` for speed 5 units.  JUMPJET it speed 7, which would imply `56 * 7 / 5 = 78`.  If you got the 71 figure directly from framerate calculations, then do we need to fix the `56` instead?